### PR TITLE
add dcs zone id to IdentityZoneConfig, use in publicKeyProvider,  and…

### PR DESCRIPTION
**Pull request by Bitty9**
… tests

https://rally1.rallydev.com/#/59288075112d/detail/userstory/166766343116
Signed-off-by: Henry <henry.zhao1@ge.com>

### CODE REVIEW

File: server/src/test/java/org/cloudfoundry/identity/uaa/provider/token/MockKeyProvider.java [hunk 1]
```diff
@@ -10,8 +10,13 @@
 
 public class MockKeyProvider implements DevicePublicKeyProvider {
 
+
     Map<String, String> publicKeys = new HashMap<>();
 
+
+    //TODO generate key
```
Comment (bsekar) [inline server/src/test/java/org/cloudfoundry/identity/uaa/provider/token/MockKeyProvider.java]: More work remaining?

**Comment by arjunnayak** [comment]
More work to be done here once DCS side of solution is available for us.

**Comment by sanjeevchopra** [comment]
this is work in progress? if yes, pls close this until its ready for review.

**Comment by ilipkind** [comment]
it is still work in progress, closing

**Pull request closed without merging**